src/main/java/formflow/library/data/SubmissionRepositoryService.java

```diff
@@ -116,32 +116,38 @@ public void removeSubflowCSRF(Submission submission, String subflowName) {
      */
     public synchronized void generateAndSetUniqueShortCode(Submission submission) {
 
+        if (shortCodeConfig.getConfig(submission.getFlow()) == null) {
```

Comment:
In theory this shouldn't happen since we null check in the ScreenController in the places this is called yea?

Reply:
Yeah, I just wanted to be sure that if someone ever used it elsewhere, we were protected against a mistake!